pkg/storage/ingest/config.go

```diff
@@ -71,6 +71,9 @@ type KafkaConfig struct {
 	WriteTimeout time.Duration `yaml:"write_timeout"`
 	WriteClients int           `yaml:"write_clients"`
 
+	SASLUsername string `yaml:"sasl_username"`
+	SASLPassword string `yaml:"sasl_password"`
```

Comment:
IIRC we had a flagext.Secret type so when this is exported the string isn't visible

Reply:
Oh very good point. I forgot about it.

Reply:
Done in https://github.com/grafana/mimir/pull/9584/commits/1b51166f8d35f5f67bbf8c575b999e03ce0276db